src/config/i18n.ts

```diff
@@ -1,14 +1,9 @@
-import { Catalog, setupI18n } from "@lingui/core";
+import { i18n } from "@lingui/core";
 
 import en from "locales/en/messages";
 import es from "locales/es/messages";
 
-const catalogs: { [key: string]: Catalog } = {
-  en,
-  es,
-};
-
-export const i18n = setupI18n({
-  language: "en",
-  catalogs,
+i18n.load({
```

Comment:
See '[i18n.load loads a catalog for a single locale](https://lingui.js.org/releases/migration-3.html#i18n-load-loads-a-catalog-for-a-single-locale)'